### PR TITLE
Fix for queries of string values

### PR DESCRIPTION
**Pull request by vmidyllic**

### CODE REVIEW

File: src/proof/proof-service.ts [hunk 1]
```diff
@@ -756,24 +780,28 @@ export class ProofService implements IProofService {
...
+    return { query, fieldName, rawValue: value };
+  }
+
+  async transformQueryValueToBigInts(value: unknown, ldType: string): Promise<bigint[]> {
+    const values: bigint[] = new Array<bigint>(64).fill(BigInt(0));
+
     if (Array.isArray(value)) {
       for (let index = 0; index < value.length; index++) {
-        values[index] = BigInt(value[index]);
+        values[index] = await Merklizer.hashValue(ldType, value);
```
Comment (nedgar) [inline src/proof/proof-service.ts:799]: @vmidyllic shouldn't this convert `value[index]`?

File: tests/proofs/sig.test.ts [hunk 2]
```diff
@@ -284,4 +284,92 @@ describe('sig proofs', () => {
...
+      ipfsGatewayURL: 'https://ipfs.io'
+    });
+
+    await credWallet.save(issuerCred);
+
+    const creds = await credWallet.findByQuery(req.body.scope[0].query);
+    expect(creds.length).to.not.equal(0);
+
+    const credsForMyUserDID = await credWallet.filterByCredentialSubject(creds, userDID);
+    expect(creds.length).to.equal(1);
```
Comment (nedgar) [inline tests/proofs/sig.test.ts:364]: Copypasta? `creds` → `credsForMyUserDID`

File: src/proof/proof-service.ts (hunk 1 above)
Comment (vmidyllic) [reply src/proof/proof-service.ts:799]: thank you. Fixed in 1.0.0-beta.15

**Comment by vmidyllic** [comment]
@nedgar fixed in #93. Expiration date will updated in next PRs.

**Pull request merged**